### PR TITLE
AutoRegisteringObjectGraphType: Inherited methods migration note

### PR DESCRIPTION
**Pull request by Shane32**
Add migration note for #3205

### CODE REVIEW

**Comment by Shane32** [comment]
cc @psmolinsky

File: docs2/site/docs/migrations/migration6.md [hunk 1]
```diff
@@ -54,6 +54,10 @@ Field<IntGraphType>("id").WithComplexityImpact(123);
 
 For more details, please review the PR here: https://github.com/graphql-dotnet/graphql-dotnet/pull/3159
 
+### 6. `AutoRegisteringObjectGraphType` recognizes inherited methods
+
+Inherited methods are now recognized by `AutoRegisteringObjectGraphType` and fields are built for them.
```
Comment (sungam3r) [inline docs2/site/docs/migrations/migration6.md:59]: 🤔 Does it make sense to do the same for inputs?

**Review by sungam3r — APPROVED** [review]

File: docs2/site/docs/migrations/migration6.md (hunk 1 above)
Comment (Shane32) [reply docs2/site/docs/migrations/migration6.md:59]: Methods cannot be written to, and so are ignored by the auto input type. Inherited properties already work for both input and output types.

Comment (sungam3r) [reply docs2/site/docs/migrations/migration6.md:59]: ok

**Pull request merged**